sklearn/model_selection/_validation.py

```diff
@@ -1500,6 +1500,32 @@ def learning_curve(
     -----
     See :ref:`examples/model_selection/plot_learning_curve.py
     <sphx_glr_auto_examples_model_selection_plot_learning_curve.py>`
+
+    Examples
+    --------
+    >>> from sklearn.datasets import make_classification
+    >>> from sklearn.tree import DecisionTreeClassifier
+    >>> from sklearn.model_selection import learning_curve
+    >>> X, y = make_classification(n_samples=100, n_features=10, random_state=42)
+    >>> tree = DecisionTreeClassifier(max_depth=4, random_state=42)
+    >>> train_size_abs, train_scores, test_scores = learning_curve(
+    ...     tree, X, y, train_sizes=[0.3, 0.6, 0.9]
+    ... )
+    >>> for train_size, cv_train_scores, cv_test_scores in zip(
+    ...     train_size_abs, train_scores, test_scores
+    ... ):
+    ...     print(f"{train_size} samples were used to train the model")
+    ...     print(f"The average train accuracy is {cv_train_scores.mean()}")
+    ...     print(f"The average test accuracy is {cv_test_scores.mean()}")
```

Comment:
I'm always concerned with showing floating points in docstring tests. To be safe, how to do feel about fixing the precision?

```suggestion
    ...     print(f"The average train accuracy is {cv_train_scores.mean():.2f}")
    ...     print(f"The average test accuracy is {cv_test_scores.mean():.2f}")
```